whats_fresh/whats_fresh_api/templates/vendor.html

```diff
@@ -256,23 +258,80 @@ <h3 class="title">Products</h3>
 
     function setAddressFields()          // For determining the location parameters.
     {
-
         $('input[name="street"]').val($('#map').data("locationpicker").addressComponents.addressLine1);
         $('input[name="city"]').val($('#map').data("locationpicker").addressComponents.city);
         $('input[name="state"]').val($('#map').data("locationpicker").addressComponents.stateOrProvince);
         $('input[name="zip"]').val($('#map').data("locationpicker").addressComponents.postalCode);
-
     }
 
-
     var preparation_options = {{ json_preparations|safe }};
 
     number_of_products = 0;
 
+    function popitup(url) {
```

Comment:
Rename this to something more explanatory, like ``createPopup()``, and I would also rename ``#dialog-form`` to something more explanatory, since it's actually the div that the popup is created in. Perhaps ``popup``? I don't think that id is used anywhere.